.github/workflows/check-pr.yml

```diff
@@ -14,11 +14,11 @@ jobs:
         uses: actions/checkout@v3
       - name: Fetch main branch
         run: git fetch origin main --depth 1
-      - uses: lots0logs/gh-action-get-changed-files@2.1.4
+      - uses: tj-actions/changed-files@v35
         id: changed-files
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The action `tj-actions/changed-files` does not appear to need the token (there is no input parameter of that kind), so this can be removed.